lib/menu.c

```diff
@@ -373,6 +376,26 @@ bm_menu_set_monitor(struct bm_menu *menu, uint32_t monitor)
         menu->renderer->api.set_monitor(menu, monitor);
 }
 
+void
+bm_menu_set_monitor_name(struct bm_menu *menu, char *monitor_name)
+{
+	assert(menu);
+
+	if (monitor_name == NULL)
+		return;
+
+	if (menu->monitor_name != NULL) {
```

Comment:
Drop the `!= NULL` to keep the codebase consistent

Reply:
Also use `&& !strcmp(menu->monitor_name, monitor_name)` and you can drop the additional branch below.